### PR TITLE
Some file path-ignore in Java CI with Maven

### PR DESCRIPTION
**Pull request by shivam-sharma7**
Some file path-ignore in Java CI with Maven

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,21 +3,33 @@ name: Java CI with Maven
...
   pull_request:
     branches: [ master ]
+    paths-ignore:
+      - '.gitignore'
+      - 'doc/**'
+      - '*.md'
+      - '*.txt'
   workflow_dispatch:
 
+    workflow_dispatch:
```
Comment (jesperpedersen) [inline .github/workflows/main.yml:21]: This is double, right ?

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (shivam-sharma7) [reply .github/workflows/main.yml:20]: Right.

Comment (shivam-sharma7) [reply .github/workflows/main.yml:20]: Let me Fixed that.

**Comment by jesperpedersen** [comment]
Rebased, and merged.

Thanks for your contribution !

**Pull request merged**